vast/eval/eval.py

```diff
@@ -1,5 +1,5 @@
 import torch
-from vast.tools import device, set_device_cpu
+from ..tools import device, set_device_cpu
```

Comment:
Thanks! I had posted a question on a similar change https://github.com/Vastlab/vast/pull/47#discussion_r753429801 but it probably slipped through your notifications.
I also found that PEP8 recommends absolute imports. For my understanding please let me know your thoughts about the linked comment.